profiles/common/modules/features/newsletters/newsletters.install

```diff
@@ -19,10 +21,13 @@ function newsletters_enable() {
 }
 
 /**
- * Use soft config for newsletters comment form location.
+ * Use soft config for comment form location and simplenews content type.
  */
 function newsletters_install() {
-  multisite_config_service('comment')->setThreadingCommentForContentType('simplenews', 1);
+  // Activate entity translation for simplenews content type.
+  variable_set('language_content_type_simplenews', ENTITY_TRANSLATION_ENABLED);
```

Comment:
I don't like the idea of forcing ET at the install of the feature. What if someone wants to use it and does not want to use entity translation ? 42 sites (at least) have disabled ET